minigalaxy/launcher.py

```diff
@@ -65,6 +65,12 @@ def determine_launcher_type(files):
         launcher_type = "final_resort"
     return launcher_type
 
+def get_exe_cmd_with_var_command(game, exe_cmd):
+    command_list = game.get_info("command").split()
+    var_list = game.get_info("variable").split()
+    var_list.insert(0, "env") # env must be still in the 1st place
```

Comment:
only if var_list is not empty.

Do you know if "env" needs to placed before every variable or only once at the beginning?
We should also check if user didn't put "env" here already and don't add "env" if it is already present.

Reply:
env needs to be placed once at the beginning (env VAR1 VAR2 VAR3 etc....)